packages/grid/_modules_/grid/components/cell/GridEditSelectCell.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import Select, { SelectProps } from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import { GridCellParams } from '../../models/params/gridCellParams';
+import { GRID_CELL_EDIT_PROPS_CHANGE_COMMITTED } from '../../constants/eventsConstants';
+
+export function GridEditSelectCell(props: GridCellParams & SelectProps) {
+  const {
+    id,
+    value,
+    formattedValue,
+    api,
+    field,
+    row,
+    colDef,
+    cellMode,
+    isEditable,
+    className,
+    getValue,
+    hasFocus,
+    ...other
+  } = props;
+
+  const handleChange = React.useCallback(
+    (event) => {
+      const newValue = event.target.value;
+      const editProps = { value: newValue };
+
+      api.publishEvent(
+        GRID_CELL_EDIT_PROPS_CHANGE_COMMITTED,
+        { id, field, props: editProps },
+        event,
+      );
+      api.setCellMode(id, field, 'view');
+    },
+    [api, field, id],
+  );
+
+  const handleClose = React.useCallback(
+    (event, reason) => {
+      if (reason === 'backdropClick') {
+        api.setCellMode(id, field, 'view');
+      }
+    },
+    [api, field, id],
+  );
+
+  return (
+    <Select
```

Comment:
Do we want to implement this type with a dropdown list? Airtable uses an combobox even for the single select.

Another concern, the demo used in the issue is a country selector, this use case is not supported by the Select, or barely (too slow).

Reply:
No opinion here - whatever we choose for the default these column types can be extended.

Reply:
The main issue for me is the upfront bundle size we impose on the developers. It seems KO to force the bundling of a column type that the developers will never use: a Select, à DatePicker, a TimePicker, an Autocomplete, etc. Before you know it, your bundle is 100 kB gzipped. Tree shaking is not helping.

In #18, I talked about how we over bundle from the core components.

How about we make this an optional colum type developers have to manually import? And then, use the Autocomplete, not the Select?

Reply:
Select is already included in the bundle because of the pagination. But manually importing this column type may be a start.

+1 for using Autocomplete. We would be able to also use it in the filter panel.

Reply:
Idk, I think it looks a bit weird if you have autocompleted with 2 options. 

Also to be clear - @oliviertassinari by not bundling you mean that if devs want to use the `select` they will load it from the grid package and spread it in the `ColDef` for that column right?

Reply:
Honestly, it's hard to see what we should prioritize here. What seems clearer is that:

- A rich date column type should be tree-shakeable. It's not OK to add 20kB? for our date picker.
- We can use `<Select>` built-in as it will always be here for the pagination
- `<Select>` doesn't work for a country select, but we aim to fix this in the core
- If we add Autocomplete in the filter, we add 5kb gzipped for everybody. 

How about we start simple, keeping the current tradeoff of the PR?

Reply:
> How about we start simple, keeping the current tradeoff of the PR?

I would go on this direction too. We use more the native select because of the ClickAwayListener issue, once the new SelectField is released we can start using it on this column type, so for developers will be clearer and acceptable (I hope so) the increase in the bundle size.

Reply:
So we can have the initial version as it is.